.github/workflows/ci-slow.yml

```diff
@@ -193,22 +193,48 @@ jobs:
           bash scripts/check-alembic-branches.sh
       - name: Install latest dashboard (test gitignore)
         run: bash scripts/install-dashboard.sh
+  ubuntu-linting:
+    needs: run-slow-ci-label-is-set
+    if: github.event.pull_request.draft == false
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
Why no `3.12` here? Since it is in fast already?

Reply:
because this is slow-ci and we have already tested it in fast-ci no need to redo